nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/NettyEventSenderFactory.java

```diff
@@ -61,6 +61,8 @@
 
     private Duration timeout = Duration.ofSeconds(30);
 
+    private Duration idleTimeout = Duration.ofSeconds(30);
```

Comment:
It seems like the internal default should be `0` given that no idle timeout is the current internal behavior of this class.

Reply:
Sure that makes sense, 30 seconds did seem arbitrary given the idle timeout default is now 15 seconds.